YOLayout/YOLayout.h

```diff
@@ -17,25 +17,25 @@ typedef enum {
   YOLayoutOptionsSizeToFitHorizontal = 1 << 1,
   YOLayoutOptionsSizeToFit = YOLayoutOptionsSizeToFitVertical | YOLayoutOptionsSizeToFitHorizontal,
 
-  // Constrain sizeThatFit's width to the pased in frame
+  // Constrain sizeThatFit's width to the passed in frame
   // For example UILabel sizeToFit may return a larger width than was specified,
   // and this will constrain it.
-  YOLayoutOptionsSizeToFitConstrainWidth = 1 << 2,
+  YOLayoutOptionsConstrainWidth = 1 << 2,
```

Comment:
I think this is the right thing to do.

I originally I left the `SizeToFit` part of these options because they only make sense when sizing to fit. But thinking about it more, most of the `YOLayoutOption`s only make sense when sizing to fit (including the `Align` options --unless you're using `setFrame:inRect:`), so I think you're right about dropping the `SizeToFit` here.